rmgpy/kinetics/surface.pyx

```diff
@@ -58,17 +58,22 @@ cdef class StickingCoefficient(KineticsModel):
     `Tmax`          The maximum temperature at which the model is valid, or zero if unknown or undefined
     `Pmin`          The minimum pressure at which the model is valid, or zero if unknown or undefined
     `Pmax`          The maximum pressure at which the model is valid, or zero if unknown or undefined
+    `coverage_dependence`      A dictionary of coverage dependent parameters to a certain surface species, with a dictionary of
```

Comment:
I think this will screw up the table in the documentation. See [here](http://rst.ninjs.org/#ICAgID09PT09PT09PT09PT09PSA9PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09CiAgICBBdHRyaWJ1dGUgICAgICAgRGVzY3JpcHRpb24KICAgID09PT09PT09PT09PT09PSA9PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09CiAgICBgQWAgICAgICAgICAgICAgVGhlIHByZWV4cG9uZW50aWFsIGZhY3RvcgogICAgYG5gICAgICAgICAgICAgIFRoZSB0ZW1wZXJhdHVyZSBleHBvbmVudAogICAgYGFscGhhYCAgICAgICAgIFRoZSBFdmFucy1Qb2xhbnlpIHNsb3BlCiAgICBgRTBgICAgICAgICAgICAgVGhlIGFjdGl2YXRpb24gZW5lcmd5IGZvciBhIHRoZXJtb25ldXRyYWwgcmVhY3Rpb24KICAgIGBUbWluYCAgICAgICAgICBUaGUgbWluaW11bSB0ZW1wZXJhdHVyZSBhdCB3aGljaCB0aGUgbW9kZWwgaXMgdmFsaWQsIG9yIHplcm8gaWYgdW5rbm93biBvciB1bmRlZmluZWQKICAgIGBUbWF4YCAgICAgICAgICBUaGUgbWF4aW11bSB0ZW1wZXJhdHVyZSBhdCB3aGljaCB0aGUgbW9kZWwgaXMgdmFsaWQsIG9yIHplcm8gaWYgdW5rbm93biBvciB1bmRlZmluZWQKICAgIGBQbWluYCAgICAgICAgICBUaGUgbWluaW11bSBwcmVzc3VyZSBhdCB3aGljaCB0aGUgbW9kZWwgaXMgdmFsaWQsIG9yIHplcm8gaWYgdW5rbm93biBvciB1bmRlZmluZWQKICAgIGBQbWF4YCAgICAgICAgICBUaGUgbWF4aW11bSBwcmVzc3VyZSBhdCB3aGljaCB0aGUgbW9kZWwgaXMgdmFsaWQsIG9yIHplcm8gaWYgdW5rbm93biBvciB1bmRlZmluZWQKICAgIGBjb3ZlcmFnZV9kZXBlbmRlbmNlYCAgICAgIEEgZGljdGlvbmFyeSBvZiBjb3ZlcmFnZSBkZXBlbmRlbnQgcGFyYW1ldGVycyB0byBhIGNlcnRhaW4gc3VyZmFjZSBzcGVjaWVzLCB3aXRoIGEgZGljdGlvbmFyeSBvZgogICAgICAgICAgICAgICAgICAgIHRoZSBhY3RpdmF0aW9uIGVuZXJneSBkZXBlbmRlbmNlIG9uIGNvdmVyYWdlLCB0aGUgcG93ZXItbGF3IGV4cG9uZW50IG9mIGNvdmVyYWdlIGRlcGVuZGVuY2UsCiAgICAgICAgICAgICAgICAgICAgYW5kIHRoZSBjb2VmZmljaWVudCBmb3IgZXhwb25lbnRpYWwgZGVwZW5kZW5jZSBvbiB0aGUgY292ZXJhZ2UKICAgIGBjb21tZW50YCAgICAgICBJbmZvcm1hdGlvbiBhYm91dCB0aGUgbW9kZWwgKGUuZy4gaXRzIHNvdXJjZSkKICAgID09PT09PT09PT09PT09PSA9PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09PT09).